hudi-utilities/src/main/java/org/apache/hudi/utilities/transform/ChainedTransformer.java

```diff
@@ -109,6 +131,28 @@ private void validateIdentifier(String id, Set<String> identifiers, String confi
     }
   }
 
+  private Option<StructType> validateAndGetTransformedSchema(TransformerInfo transformerInfo, Dataset<Row> dataset, Option<StructType> incomingStructOpt,
+                                                             JavaSparkContext jsc, SparkSession sparkSession, TypedProperties properties) {
+    if (!incomingStructOpt.isPresent()) {
```

Comment:
nit: you can use `orElseThrow` here as a cleaner way of doing something like this.

`StructType incomingStruct = incomingStructOpt.orElseThrow(() -> new HoodieSchemaException(String.format("Source schema not available for transformer %s", transformerInfo));`

Reply:
Removed Option from tranformedSchema API so removed this code